CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# On Github Issues and Pull Requests
+
+Found a bug? Want to contribute changes to the codebase? Make sure to read this first.
```

Comment:
What should the distinction be for an example being merged here vs. keras proper? Obviously, using functionality from contrib should necessitate it's presence in contrib, but how about underdeveloped or overly niche examples?